src/Whim.FloatingLayout/FloatingLayoutEngine.cs

```diff
@@ -1,152 +1,129 @@
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using System.Collections.Immutable;
 using System.Linq;
 
 namespace Whim.FloatingLayout;
 
 /// <summary>
-/// A proxy layout engine to allow windows to be free-floating.
+/// Layout engine that lays out all windows as free-floating.
```

Comment:
This going to be a pain in the arse but I think it should be a part of this PR:

1. rename `src/Whim.FloatingLayout` and `src/Whim.FloatingLayout.Tests` to `src/Whim.FloatingWindow` etc.
2. rename the `Whim.FloatingLayout` namespace to `Whim.FloatingWindow`
3. update the solution file `Whim.sln` to use `Whim.FloatingWindow` instead of `Whim.FloatingLayout`
4. update the `Whim.Runner.csproj` similarly
5. update the csx with the updated reference and import `using`

If this proves problematic I can tack this onto this PR if you wish.

Reply:
Done